python/orca/src/bigdl/orca/common.py

```diff
@@ -158,22 +158,21 @@ def get_ray_context():
         return RayContext.get()
 
 
-def init_orca_context(cluster_mode="local", cores=2, memory="2g", num_nodes=1,
+def init_orca_context(cluster_mode=None, cores=2, memory="2g", num_nodes=1,
                       init_ray_on_spark=False, **kwargs):
     """
     Creates or gets a SparkContext for different Spark cluster modes (and launch Ray services
     across the cluster if necessary).
-
     :param cluster_mode: The mode for the Spark cluster. One of "local", "yarn-client",
            "yarn-cluster", "k8s-client", "standalone" and "spark-submit". Default to be "local".
-
            For "spark-submit", you are supposed to use spark-submit to submit the application.
            In this case, please set the Spark configurations through command line options or
            the properties file. You need to use "spark-submit" for yarn-cluster or k8s-cluster mode.
+           For "k8s-client", you are supposed to additionally specify the arguments master 
+           and container_image.
```

Comment:
Put this before For spark-submit and add a blank line, since the next paragraph goes with spark-submit